elastio-ec2-backup_ecs_stopped_task/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# syntax=docker/dockerfile:1
+
+# Version elastio cli what will use like base image.
+# Most likely this is not the latest version of elastio cli.
+# You can find the latest version at this link.
+# https://gallery.ecr.aws/elastio-dev/elastio-cli
+FROM public.ecr.aws/elastio-dev/elastio-cli:0.17.0-master
+
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get install -y python3-pip python3
+COPY requirements.txt  ./
+RUN python3 -m pip install --no-cache-dir -r requirements.txt
+COPY failed_ecs_handler.py ./
+COPY failed_ecs_handler_config.py ./
```

Comment:
This is not a very efficient way to build the image.  You should copy the files first, then have a single `RUN` which uses `&&` to combine the operations, and at the and you need to clean up the `apt-get` cache so that it isn't part of the container image and wasting space unncessarily.  This is a standard pattern; I don't recall the exact command to clean up the apt cache but you'll see it everywhere.  Maybe @ivankovnatsky can point you to another example where this is done.